omero/sysadmins/repository-move.txt

```diff
@@ -117,3 +117,37 @@ to ``/Volumes/imports/ManagedRepository``:
     If :property:`omero.managed.dir` is set then the Managed Repository and
     the OMERO data directory should be treated independently and thus be moved
     separately if necessary.
+
+Extending the Managed Repository
+--------------------------------
+
+It is possible to leave the Managed Repository in place yet have newly
+imported image files stored on a different underlying storage volume.
+For example, if your :property:`omero.managed.dir` is set to
+:file:`/mnt/omero/ManagedRepository` and, as it fills, it would be
+better for future images to be imported onto
+:file:`/mnt/omero/large-volume-B/` then an OMERO administrator may use
+the :omerocmd:`fs mkdir` subcommand to properly set up a subdirectory in
+the existing Managed Repository:
+::
+
+    bin/omero fs mkdir volume-B
```

Comment:
If you've just read the previous sentence: "If...better.... /mnt/omero/large-volume-B/ ...then...." ```bin/omero fs mkdir volume-B``` is confusing. It only makes sense after the next symlink paragraph (if you choose to symlink).
I only noticed this difference having read the last section "new imports should upload onto
 +:file:`/mnt/omero/large-volume-B/`  " and had missed all previous mentions of "large" since I didn't notice it the first time I read the section above and I missed it in the ```ln -s``` command. 
Maybe the easiest fix is to use "large-volume-B" everywhere or just use "volume-B" everywhere? Then all sections make sense whether you choose to symlink or not?